plasmapy/formulary/collisions.py

```diff
@@ -2284,7 +2295,7 @@ def Lorentz_collision_frequency(self):
         (typically taken as the thermal velocity), and :math:`\ln{Λ}` is the
```

Comment:
```suggestion
         and :math:`\ln{Λ}` is the
```

I think we should drop this part about the thermal velocity here, since for many uses of this function v_drift is going to be set much higher than the thermal velocity.